lib/pdcleaner/pdcleaner_test.go

```diff
@@ -0,0 +1,204 @@
+package pdcleaner
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strconv"
+	"testing"
+
+	bdb "github.com/filecoin-project/boost/db"
+	"github.com/filecoin-project/boost/db/migrations"
+	"github.com/filecoin-project/boost/extern/boostd-data/client"
+	"github.com/filecoin-project/boost/extern/boostd-data/model"
+	"github.com/filecoin-project/boost/extern/boostd-data/svc"
+	mocks_legacy "github.com/filecoin-project/boost/lib/legacy/mocks"
+	"github.com/filecoin-project/boost/piecedirectory"
+	"github.com/filecoin-project/boost/storagemarket/types/dealcheckpoints"
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/go-state-types/abi"
+	"github.com/filecoin-project/go-state-types/builtin/v9/market"
+	"github.com/filecoin-project/lotus/api"
+	lotusmocks "github.com/filecoin-project/lotus/api/mocks"
+	test "github.com/filecoin-project/lotus/chain/events/state/mock"
+	chaintypes "github.com/filecoin-project/lotus/chain/types"
+	"github.com/golang/mock/gomock"
+	"github.com/google/uuid"
+	"github.com/ipfs/go-cid"
+	"github.com/ipld/go-car/v2"
+	"github.com/stretchr/testify/require"
+)
+
+func TestPieceDirectoryCleaner(t *testing.T) {
+	req := require.New(t)
+	ctx := context.Background()
+
+	provAddr, err := address.NewIDAddress(1523)
+	require.NoError(t, err)
```

Comment:
Use `req` consistently or remove its allocation and stick with package level exported testify library?